test/powershell/enginecore/CommandDiscovery.Tests.ps1

```diff
@@ -0,0 +1,105 @@
+﻿Describe "Command Discovery tests" -Tags "CI" {
+
+    BeforeAll {
+        setup -f testscript.ps1 -content "TestScript"
```

Comment:
Fixed

---
In reply to: [79887091](https://github.com/PowerShell/PowerShell/pull/2285#discussion_r79887091) [](ancestors = 79887091)